src/containers/listView/containerCellEditor.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import Input from '@material-ui/core/Input';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+
+export class ContainerCellEditor extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            containerId: (this.props.value) ? this.props.value._id : "",
+            open: false
+        }
+        // this.props.value is the currently selected value
```

Comment:
Do you want to rename this.props.value to something like this.props.currentValue to make it more clear, based on this comment?

Reply:
I can't name them. They are named by the third party component Ag-Grid. This is just a note for myself.